Examples/Python/src/Geometry.cpp

```diff
@@ -183,13 +183,18 @@ void addExperimentalGeometry(Context& ctx) {
 
   using namespace Acts::Experimental;
 
-  // Detector definition
-  py::class_<Detector, std::shared_ptr<Detector>>(m, "Detector");
-
   // Detector volume definition
   py::class_<DetectorVolume, std::shared_ptr<DetectorVolume>>(m,
                                                               "DetectorVolume");
 
+  // Detector definition
+  py::class_<Detector, std::shared_ptr<Detector>>(m, "Detector")
+      .def("number_volumes",
```

Comment:
Could be a property

Reply:
Don't know what this means ... a python property of the detector?